fetch/api/redirect/redirect-count-cross-origin.js

```diff
@@ -0,0 +1,59 @@
+if (this.document === undefined) {
+  importScripts("/resources/testharness.js");
+  importScripts("../resources/utils.js");
+  importScripts("/common/utils.js");
+  importScripts("/common/get-host-info.sub.js");
+}
+
+function redirectCount(desc, redirectUrl, redirectLocation, redirectStatus, maxCount1, maxCount2, requirePreflight)
```

Comment:
Maybe you're misunderstanding my feedback. My point was that they don't have to be arguments to the `redirectCount()` function as the `redirectCount()` function is never invoked with different values for them. Surely `redirect-count.py` is not invoking JavaScript?